lib/core/managementClient.ts

```diff
@@ -26,6 +28,79 @@ export enum DispositionStatus {
   renewed = "renewed"
 }
 
+/**
+ * Represents a description of a rule.
+ */
+export interface RuleDescription {
+  /**
+   * Filter expression used to match messages.
+   */
+  filter?: SQLExpression | CorrelationFilter;
+  /**
+   * Action to perform if the message satisfies the filtering expression.
+   */
+  action?: SQLExpression;
+  /**
+   * Represents the name of the rule.
+   */
+  name: string;
+}
+/**
+ * Represents the sql filter expression.
+ */
+export interface SQLExpression {
+  /**
+   * SQL-like condition expression that is evaluated in the broker against the arriving messages'
+   * user-defined properties and system properties. All system properties must be prefixed with
+   * `sys.` in the condition expression.
+   */
+  expression: string;
+}
+
+/**
+ * Represents the correlation filter expression.
+ * A CorrelationFilter holds a set of conditions that are matched against one of more of an 
+ * arriving message's user and system properties.
+ */
+export interface CorrelationFilter {
```

Comment:
We should export all these interfaces from lib/index.ts, so customers can use them for type declaration if required.

Reply:
Done